### PR TITLE
Added published email content.

### PR DESCRIPTION
**Pull request by waheedahmed**
[ECOM-6083](https://openedx.atlassian.net/browse/ECOM-6083)

As a Publisher tool owner, I'd like an email template created which will be used to send emails to Course Team when a Course-run has been published.

screen shot 2017-03-02 at 6 37 29 pm

### CODE REVIEW

File: course_discovery/apps/publisher/api/tests/test_views.py [hunk 1]
```diff
@@ -681,6 +682,8 @@ def test_course_published(self):
         self.run_state.name = CourseRunStateChoices.Approved
         self.run_state.preview_accepted = True
         self.run_state.save()
+        self.course_run.lms_course_id = 'course-v1:testX+test45+2017T2'
```
Comment (bderusha) [inline course_discovery/apps/publisher/api/tests/test_views.py]: Will it break other test cases to add lms_course_id in the setup method?

File: course_discovery/templates/publisher/email/course_run/published.html [hunk 2]
```diff
@@ -8,13 +8,13 @@
...
         {% blocktrans with link_start='<a href="' link_middle='">' link_end='</a>' trimmed %}
-            Your course {{ link_start }}{{ page_url }}{{ link_middle }} {{ course_name }} {{ link_end }} is now live.
+            The {{ link_start }}{{ page_url }}{{ link_middle }} {{ course_run_number }} course run {{ link_end }} of {{ course_name }} has been published. Visit {{ link_start }}{{ preview_url }}{{ link_middle }}link to live course run page{{ link_end }} to view the course run.
         {% endblocktrans %}
     </p>
 
     {% comment %}Translators: It's closing of mail.{% endcomment %}
     {% trans "Thanks," %}<br>
-    {% trans "The edX team" %}
+    edX {{ sender_role }}
```
Comment (bderusha) [inline course_discovery/templates/publisher/email/course_run/published.html]: Just noticing this now, but shouldn't `edX` also be dynamic for platforms which are not edX?

File: course_discovery/apps/publisher/api/tests/test_views.py (hunk 1 above)
Comment (waheedahmed) [reply course_discovery/apps/publisher/api/tests/test_views.py]: @bderusha No, its only being used in published email to extract the run from it.

File: course_discovery/templates/publisher/email/course_run/published.html (hunk 2 above)
Comment (waheedahmed) [reply course_discovery/templates/publisher/email/course_run/published.html]: @bderusha Yeah it should be, but not mentioned in any ticket.

**Comment by waheedahmed** [comment]
@bderusha Feedback addressed, please take another look.

**Review by tasawernawaz — APPROVED** [review]
LGTM

File: course_discovery/apps/publisher/api/tests/test_views.py (hunk 1 above)
Comment (bderusha) [reply course_discovery/apps/publisher/api/tests/test_views.py]: Then, can we move this into the setup of the course_run?

**Review by bderusha — APPROVED** [review]

File: course_discovery/apps/publisher/api/tests/test_views.py (hunk 1 above)
Comment (waheedahmed) [reply course_discovery/apps/publisher/api/tests/test_views.py]: Actually I just realized its already added in setup recently, so we don't need it here now.

**Pull request merged**